Automattic/wp-calypso
# Proof of Concept: Use IsolatedBlockEditor to edit a post

#### Changes proposed in this Pull Request

* Use the `IsolatedBlockEditor` to edit a full post

#### Testing instructions



* Go to `/without-iframe/post/:site/:postId` and see that a the post is loaded and editable using the isolated block editor (you cannot save though)

Fixes #
